wrn-fix-it.py

```diff
@@ -0,0 +1,381 @@
+            # Quad9 DNS
+            t001_d0 = "0"
+            t001_d1 = "9.9.9.9"
+            t001_d2 = "149.112.112.112"
+            t001_1(t001_d0, t001_d1, t001_d2)
+            break
+        elif user_input == "6":
+            # Verisign DNS
+            t001_d0 = "0"
+            t001_d1 = "64.6.64.6"
+            t001_d2 = "64.6.65.7"
+            t001_1(t001_d0, t001_d1, t001_d2)
+            break
+        elif user_input == "7":
+            # OpenDNS
+            t001_d0 = "0"
+            t001_d1 = "208.67.222.222"
+            t001_d2 = "208.67.220.220"
+            t001_1(t001_d0, t001_d1, t001_d2)
+            break
+        elif user_input == "8" or user_input == "":
+            tools_menu()
+            break
+        else:
+            print("Invalid input. Please try again.")
+            continue
+
+
+def tools_menu():
+    """Select tools for configuring networks"""
+
+    clear()
+    print(f"# {divider}")
+    print(f"# {appname} v{appvers} - {appstat}")
+    print(f"# by {dev}")
+    print(f"# {divider}")
+    print(f"#")
+    print(f"# DNS Server Changer ......... [1]")
+    print(f"# Back to Main Menu .......... [2] (enter)")
+    print(f"#")
+
+    while True:
+        user_input = input("toolsMenu=# > ")
+
+        if user_input == "1":
+            t001()
+            break
+        elif user_input == "2" or user_input == "":
+            main_menu()
+            break
+        else:
+            print("Invalid input. Please try again.")
+            continue
+
+
+def modules_menu():
+    """The modules menu, WIP"""
+
+    clear()
+    print(f"# {divider}")
+    print(f"# {appname} v{appvers} - {appstat}")
+    print(f"# by {dev}")
+    print(f"# {divider}")
+    print(f"# Currently not available!")
+    print(f"# {divider}")
+    print(f"#")
+    print(f"# Press Enter to continue...")
+    input()
+    main_menu()
+
+
+def launch(url):
+    """Launch a website in the default browser
+       given a URL."""
+
+    if name == "nt":
+        system(f"start {url}")
+    elif name == "posix":
+        system(f"xdg-open {url}")
+
+
+def donate():
+    """Links to donation sites"""
+
+    clear()
+    print(f"# {divider}")
+    print(f"# {appname} v{appvers} - {appstat}")
+    print(f"# by {dev}")
+    print(f"# {divider}")
+    print(f"# Donate and Support")
+    print(f"# {divider}")
+    print(f"# ")
+    print("# Do you like this tool?")
+    print("# Please consider to buy me a coffee or")
+    print("# just donate to keep this project alive.")
+    print("#")
+    print("# Buy Me A Coffee ............ [1]")
+    print("# PayPal ..................... [2]")
+    print("# Back to Main Menu .......... [3] (enter)")
+    print("#")
+
+    while True:
+        user_input = input("donate=# > ")
+        if user_input == "1":
+            launch("https://buymeacoff.ee/warengonzaga")
+            break
+        elif user_input == "2":
+            launch("https://paypal.me/warengonzagaofficial")
```

Comment:
I guess for the python version we can safely remove this link for PayPal.
I would stick to BMC since it accepts PayPal too.

Reply:
Working on it